### PR TITLE
Restructure the form elements

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
     window.onload = getProcessList;
 
-    function showTextEditor() {
-        saveProcess();
+    function showTextEditor(e1) {
```
Comment (milindaperera) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: Better if we can use some proper variable names which depicts the variable

File: modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs [hunk 2]
```diff
@@ -15,8 +15,8 @@
         });
     }
 
-    function associateBPMN() {
-        saveProcess();
+    function associateBPMN(e2) {
```
Comment (milindaperera) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: Better if we can use some proper variable names which depicts the variable

File: modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs (hunk 1 above)
Comment (isurusuranga) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: Fixed in https://github.com/wso2/product-pc/pull/35

File: modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs (hunk 2 above)
Comment (isurusuranga) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: Fixed in https://github.com/wso2/product-pc/pull/35

**Pull request merged**